src/seedu/addressbook/Main.java

```diff
@@ -124,5 +132,10 @@ private StorageFile initializeStorage(String[] launchArgs) throws InvalidStorage
         return isStorageFileSpecifiedByUser ? new StorageFile(launchArgs[0]) : new StorageFile();
     }
 
-
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.